pages/facts.html

```diff
@@ -4,9 +4,72 @@
   <meta charset="UTF-8">
   <meta http-equiv="X-UA-Compatible" content="IE=edge">
   <meta name="viewport" content="width=device-width, initial-scale=1.0">
-  <title>Document</title>
+  <link href="css/style.css" rel="stylesheet">
+  <link href="css/facts.css" rel="stylesheet">
+  <!-- Import Google fonts: Roboto -->
+  <link rel="preconnect" href="https://fonts.googleapis.com">
+  <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
+  <link href="https://fonts.googleapis.com/css2?family=Roboto:ital,wght@0,100;0,300;0,500;0,700;0,900;1,100;1,300;1,500;1,700;1,900&display=swap" rel="stylesheet">
+  <!-- End Google fonts import -->
+  <title>Fun Facts about Cartoons</title>
 </head>
 <body>
-  
+  <header>
+    <nav>
+      <ul>
+        <li><a href="index.html">Home</a></li>
+        <li><a href="facts.html">Fun Facts</a></li>
+        <li><a href="gallery.html">Gallery</a></li>
+      </ul>
+    </nav>
+  </header>
+  <section>
+    <article>
+      <h1>Fun Facts About...</h1>
```

Comment:
It's a little subjective, but in this layout I might consider the header and the section as part of the significant content of the page and place it inside a `main` tag.